packages/mds-types/index.ts

```diff
@@ -349,9 +349,9 @@ export interface TripsStats {
 
 // The above types represent objects that can be created and passed into functions that write to the database. The
 // following type alias allows wrapping the above types with Recorded<> in order to represent what is read from the
-// database. This type alias will add the readonly attribute to all properties and also remove undefined as a valid
-// value since the database will never return undefined.
-export type Recorded<T> = Readonly<Required<T>>
+// database. This type alias will add the identity column, remove the readonly attribute from all properties, and
```

Comment:
It seems like this adds the readonly attribute, no?

Reply:
No, it also adds the `id` column, etc. 
```
export type Recorded<T> = Readonly<Required<T & { id: number }>>
```

Reply:
Yes, I see that. It _also_ *adds* the readonly attribute, however the comment dictates that it *removes* the readonly attribute. `remove the readonly attribute from all properties`. I'm saying you should update the comment to match what the behavior is